Team.cpp

```diff
@@ -0,0 +1,47 @@
+#include "Team.h"
+
+Team::Team(std::string nameNew, int fifaRangNew, double fifaPointsNew) :
```

Comment:
strings are copied, so either declare as const& or pass a string_view and copy from it.